src/pages/dataTableUtilities/getPageInitialiser.js

```diff
@@ -365,6 +365,18 @@ const supplierRequisitionInitialiser = requisition => {
       ? sortedData
       : sortedData.filter(item => item.isLessThanThresholdMOS);
 
+  const indicators = program?.indicators?.slice() || [];
+  const [selectedIndicator] = indicators;
+
+  const indicatorColumns = selectedIndicator.columns.map(column => ({
+    type: column.valueType,
+    key: column.description,
+    title: column.description,
+    width: 10,
+    sortable: false,
+    editable: false,
```

Comment:
possibly this too - is there a way to differentiate between the static, uneditable`IndicatorData` and the enterable?

Reply:
Just for now keeping these as constants, but have made a desktop issue related this.